modules/virt-creating-linux-bridge-nad-cli.adoc

```diff
@@ -50,6 +51,7 @@ spec:
 <6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
 <7> Optional: The VLAN tag. No additional VLAN configuration is required on the node network configuration policy.
 <8> Optional: Indicates whether the VM connects to the bridge through the default VLAN. The default value is `true`.
+<9> Optional: Indicates if the state of the container interface (virtual ethernet device peer inside the network namespace) is down. The default value is `true` (down). You cannot use IP Address Management (IPAM) when this field is set to `false`.
```

Comment:
The `disableContainerInterface` option default value is `false`.
When set to `true` the container intreface will be set to `down`.
Please see the commit messge https://github.com/openshift/containernetworking-plugins/pull/154/commits/d977f78ef0eb315551e0bf42de791aa9b486132c
And PR for more details https://github.com/containernetworking/plugins/pull/997

Reply:
I think it also worth noting that this setting is relevat for setups that use IPv6, dual-stack of single stack IPv6 (not sure if GA yet)